packages/datadog-instrumentations/src/jest.js

```diff
@@ -262,7 +258,19 @@ addHook({
   })
 
   return coverageReporter
-})
+}
+
+addHook({
+  name: '@jest/reporters',
+  file: 'build/coverage_reporter.js',
+  versions: ['>=24.8.0']
```

Comment:
Wouldn't that fail starting with 26.6.2?

Reply:
yeah good catch, added a version limit